modules/claims_api/config/schemas/526.json

```diff
@@ -1008,7 +1008,31 @@
           {
             "if": {
               "properties": {
-                "disabilityActiontype": {
+                "disabilityActionType": {
+                  "enum": ["NEW"]
+                }
+              },
+              "dependencies": {
+                "disabilityActionType": {
+                  "not": {
+                    "required": ["classificationCode"]
+                  }
```

Comment:
[some extra context for the JSON schema logic](https://stackoverflow.com/a/61062869/5469511)